README.md

```diff
@@ -12,75 +12,7 @@ pip install yoctol-nlu
 
 ## Usage
 
-### Intent Classifier Service
-
-#### Fetch Model & One Line Prediction
```

Comment:
I think it's OK to contain this session since it's the core functionality.

Reply:
ok, gonna put it back